app/serializers/team_serializer.rb

```diff
@@ -38,7 +38,7 @@ def user
     current_user
   end
 
-  def users_favorised_current_team
-    @users_favorised_current_team ||= object.user_favourite_teams.pluck(:team_id)
+  def user_favourite_team_ids
+    @user_favourite_team_ids ||= object.user_favourite_teams.pluck(:team_id)
```

Comment:
this does not work ... this return all favourite team entries from all users !
you need to get favourite teams from current user